open-hackathon-server/src/hackathon/hack/hackathon_manager.py

```diff
@@ -715,34 +793,36 @@ def check_hackathon_online(self, hackathon):
 
         return ok(can_online)
 
+    # TODO: some information is omitted and needed to be added
     def __get_hackathon_detail(self, hackathon, user=None):
         """Return hackathon info as well as its details including configs, stat, organizers, like if user logon"""
         detail = hackathon.dic()
 
-        detail["config"] = self.__get_hackathon_configs(hackathon)
-        detail["stat"] = self.get_hackathon_stat(hackathon)
-        detail["tag"] = self.get_hackathon_tags(hackathon)
-        detail["organizer"] = self.__get_hackathon_organizers(hackathon)
+        # detail["config"] = self.__get_hackathon_configs(hackathon)
```

Comment:
`config`, `tags`, `organizer` and `partner` are embedded documents now in `Hackathon` model.